Dockerfile.signimage

```diff
@@ -1,24 +1,27 @@
-FROM registry.fedoraproject.org/fedora as ksource
-RUN yum install -y kernel-devel
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11 as ksource
+RUN dnf install kernel-devel
 
-FROM golang:1.19 as builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11 as builder
 
 WORKDIR /workspace
 
+# Download dependencies
+COPY go.mod go.mod
+COPY go.sum go.sum
+RUN go mod download
```

Comment:
We should not run this m/s. All dependencies are in `vendor`.

Reply:
I can remove `go mod download` but if I remove `go.mod` and `go.sum` I get:
```
go build -o signimage cmd/signimage/signimage.go
go: updates to go.mod needed, disabled by -mod=vendor; to update it:
        go mod tidy
make: *** [Makefile:271: signimage] Error 1
Error: error building at STEP "RUN make signimage": error while running runtime: exit status 2
```